tests/foreman/maintain/test_service.py

```diff
@@ -0,0 +1,309 @@
+    sat_maintain.execute(
+        f'sed -i "s/{flag}: {settings.server.admin_password}/{flag}: invalid/g" '
+        f'{satellite_answer_file}'
+    )
+    sat_maintain.execute(f'mv {HAMMER_CONFIG} /tmp/foreman.yml')
+    sat_maintain.execute(f'mv {maintain_hammer_yml} /tmp/foreman-maintain-hammer.yml')
+
+    @request.addfinalizer
+    def _finalize():
+        sat_maintain.execute(
+            f'sed -i "s/{flag}: invalid/{flag}: {settings.server.admin_password}/g" '
+            f'{satellite_answer_file}'
+        )
+        sat_maintain.execute(f'mv /tmp/foreman.yml {HAMMER_CONFIG}')
+        sat_maintain.execute(f'mv /tmp/foreman-maintain-hammer.yml {maintain_hammer_yml}')
+
+
+@pytest.mark.include_capsule
+def test_positive_service_restart(sat_maintain):
+    """Restart services using satellite-maintain service subcommand
+
+    :id: c5a38994-8c14-40c7-bc6a-1cfc68fc2d28
+
+    :parametrized: yes
+
+    :steps:
+        1. Run satellite-maintain service restart
+
+    :expectedresults: Satellite services should restart.
+    """
+    result = sat_maintain.cli.Service.restart()
+    logger.info(result.stdout)
+    assert 'FAIL' not in result.stdout
+    assert result.status == 0
+
+
+@pytest.mark.include_capsule
+def test_positive_service_list(sat_maintain):
+    """List satellite services using satellite-maintain service subcommand
+
+    :id: aa878d16-5a22-11ed-849d-cb7ac12cece2
+
+    :parametrized: yes
+
+    :steps:
+        1. Run satellite-maintain service list
+
+    :expectedresults: Satellite services should be listed
+    """
+    result = sat_maintain.cli.Service.list()
+    logger.info(result.stdout)
+    assert 'FAIL' not in result.stdout
+    assert result.status == 0
+
+
+@pytest.mark.include_capsule
+def test_positive_service_stop_start(sat_maintain):
+    """Start/Stop services using satellite-maintain service subcommand
+
+    :id: fa3b02a9-b441-413b-b9d2-1a59d04f285c
+
+    :parametrized: yes
+
+    :steps:
+        1. Run satellite-maintain service stop
+        2. Run satellite-maintain service status --brief
+        3. Run satellite-maintain service start
+        4. Run satellite-maintain service status --brief
+
+    :expectedresults: Satellite services should stop and start.
+    """
+    result = sat_maintain.cli.Service.stop()
+    logger.info(result.stdout)
+    assert 'FAIL' not in result.stdout
+    assert result.status == 0
+    result = sat_maintain.cli.Service.status(options={'brief': True})
+    logger.info(result.stdout)
+    assert 'FAIL' in result.stdout
+    assert result.status == 1
+    result = sat_maintain.cli.Service.start()
+    logger.info(result.stdout)
+    assert 'FAIL' not in result.stdout
+    assert result.status == 0
+    result = sat_maintain.cli.Service.status(options={'brief': True})
+    logger.info(result.stdout)
+    assert 'FAIL' not in result.stdout
+    assert result.status == 0
+
+
+@pytest.mark.include_capsule
+def test_positive_service_stop_restart(sat_maintain):
```

Comment:
What do you think of merging this test with https://github.com/SatelliteQE/robottelo/pull/10231/files#diff-122f6911bfcde697ff27cd2f1d73e85bf0ac0e01aacb371e7fc0c7cbbec5a181R90 ?
As only one additional step of restart is present here, can be added in above test

Reply:
@shubhamsg199 yes, we can combine this tests, and perform stop twice, but I think its better to be explicit, and keep an separate test for it